app/basicComponents/AutocompleteDropdown.tsx

```diff
@@ -288,12 +288,13 @@ type Props = {
   onChange: (value: string) => void;
   onEnter: (p: any) => void;
   autofocus?: boolean;
+  autocomplete?: boolean;
 };
 
 export type AutocompleteDropdownProps = Props;
 
 const AutocompleteDropdown = (props: Props) => {
-  const { value, autofocus } = props;
+  const { value, autofocus, autocomplete = true } = props;
```

Comment:
for the component , to be able to list all addresses and at the same time has possibility to type new reward address